### ISSUE TITLE
5.1.2 Max Pool. Mistake on Page 128?

### ISSUE THREAD

**Issue by MbProg** [fchollet/deep-learning-with-python-notebooks#103]
On this page it states about not using Max Pooling: 

> It isn’t conducive to learning a spatial hierarchy of features. The 3 × 3 windows in the third layer will only contain information coming from 7 × 7 windows in the initial input. 

I don't understand where the 7x7 window comes from. The output of the second layer is 24x24. Is that a mistake in the book?

**Comment by tabmoo** [comment]
No, the book is correct. The term 'window' here doesn't mean 'filter size'. Chollet wants to say that the information from each 7x7 'zone' of the 28x28 input image is seen only by 3x3 'zone'  in the third layer. With usage of MaxPooling layers the size of the 'zone' of the input image from which the third layer 'zone' receives information would be much larger.
